src/lib/holocene/select/select.story.svelte

```diff
@@ -79,6 +79,24 @@
     </Select>
   </Hst.Variant>
 
+  <Hst.Variant title="With Grouped Options different value and display">
+    <Select id="select-3" placeholder="Select an Option" {value} {disabled}>
+      <OptionGroup label="Mammals">
+        <Option value={1}>Horse</Option>
+        <Option value={2}>Dog</Option>
+        <Option value={3}>Cat</Option>
+      </OptionGroup>
+      <OptionGroup label="Reptiles">
+        <Option value={4}>Snake</Option>
+        <Option value={5}>Lizard</Option>
+      </OptionGroup>
+      <OptionGroup label="Amphibians">
+        <Option value={6}>Frog</Option>
+        <Option value={7}>Salamander</Option>
+      </OptionGroup>
+    </Select>
+  </Hst.Variant>
```

Comment:
Probably can 🚮

Reply:
This is essentially a test case to verify that the changes we implemented work. I think I'd prefer to keep it around.

Reply:
Oh I didn't notice these `<Option />`'s had `value`'s, sounds good 👍